src/Plivo/Util/ArrayOperations.php

```diff
@@ -17,14 +17,7 @@ class ArrayOperations
      */
     public static function removeNull(array $array)
     {
-        $result = [];
-        foreach ($array as $key => $value) {
-            if (is_null($value)) {
-                continue;
-            }
-            $result[$key] = $value;
-        }
-        return $result;
+        return array_filter($array);
```

Comment:
This removes those array elements where the values are false, 0, 0.0, [] etc., The desired effect is just to have the null values removed.